content/departments/people-talent/people-ops/process/how-we-engage-talent-outside-the-us/deel.md

```diff
@@ -0,0 +1,59 @@
+
+## FAQs
+
+#### Why is there no mention of equity or stocks in my contract?
+
+Since your employment contract is with Deel, any equity grants or stock options offered by your company will need a separate agreement between you and them.
+
+We usa Carta as our default app for tracking options, please refer to [this handbook](../../../../../benefits-pay-perks/pay-expenses/compensation/index.md#components-of-compensation) page for more information. You can also confirm your number of options and grant date in BambooHR.
+
+#### What’s the name of my employer?
+
+You can check who employs you in [this page](https://help.letsdeel.com/hc/en-gb/articles/4414100793233-What-s-the-name-of-my-employer-).
+
+#### How often does the exchange rate get updated for compensation?
+
+The exchange rate is based on a 12 month trailing average, but and it's only updated when there is a change to compensation - for example, Merit or Promotion or Band adjustment.
+
+#### What does the onboarding process look like?
+
+Do to [this page](https://help.letsdeel.com/hc/en-gb/sections/8266345839249-Onboarding-Guide) to read about the process of being onboarded in Deel.
+
+#### How can I submit an expense in Deel?
+
+You can see this information in [this handbook page](../../../../../benefits-pay-perks/pay-expenses/expenses/expenses-through-deel.md).You can find more information on how to submit expenses in Deel in[ this page](https://help.letsdeel.com/hc/en-gb/sections/8266431839377-Expenses).
+
+#### The PTO policie in my country of recidence is different from Sourcegraph’s PTO policy, which one should I follow?
+
+Sourcegraph will honor the unlimited [PTO policy](../../../../../benefits-pay-perks/benefits-perks/time-off/index.md), but you will be required to take the minimum mandatory ones for your country in order for us to be compliant with Deel and your country’s laws.
+
+#### How do I submit time off?
+
+You can check [this article](https://help.letsdeel.com/hc/en-gb/articles/4409044013201-How-do-I-request-time-off-) on how to ask for time off through Deel’s platform.
+
+Sourcegraph uses Roots to submit PTO. There is an integration set up with Roots and Deel, so you will be able to submit time off via Roots and it will automatically sync to Deel. For this to work, your Deel email will need to match your Roots email (your Sourcegraph email). Make sure you change your Deel email to your corporate one if you wish this integration to apply to you.
+
+You can read more about submitting time off via [Roots here](../../../../../benefits-pay-perks/benefits-perks/time-off/submitting-time-off.md).
+
+#### How can I get an Employment Verification Letter from Deel?
+
+Go to [this article ](https://help.letsdeel.com/hc/en-gb/articles/4906891050769-How-do-I-download-an-Employment-Verification-Letter-)for more information on this topic.
```

Comment:
If I move will I be able to switch countries and still use deel?

Reply:
FAQ suggestion: Will I be able to change my contract relationship later on if needed? (from EOR to something else?)

Reply:
Amazing! TY